raiden/raiden_event_handler.py

```diff
@@ -172,6 +173,13 @@ def handle_contract_send_secretreveal(
     secret_registry.register_secret(channel_close_event.secret)
 
 
+def handle_contract_receive_secretreveal(
+        raiden: 'RaidenService',
+        receive_secret_reveal_event: ContractReceiveSecretReveal,
+):
+    pass
```

Comment:
What should be done here? Loredana told me that we shouldn't add the known secretes to some state. Anything else or do we just want to have them in the WAL?